test/unit/dsl/control_test.rb

```diff
@@ -15,6 +15,31 @@ def load(content)
                    .params[:controls]['1']
   end
 
+  let :rand_string { rand.to_s }
+
+  it 'adds a title' do
+    load("title #{rand_string.inspect}")[:title].must_equal rand_string
+  end
+
+  it 'adds a description' do
+    load("desc #{rand_string.inspect}")[:desc].must_equal rand_string
+  end
+
+  it 'adds a multiline description' do
+    t = rand_string + "\n" + rand_string
```

Comment:
for example, this was not working with the old method, it would just replace the newline character